portfolio/src/main/java/com/google/sps/servlets/ApiKeyServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/apiKeyServlet")
```

Comment:
I think this servlet can be used in the future to deliver other key-value entries to the client. So consider making it more general, like /appConfigServlet or something. Then we can potentially use it for other config fields, in addition to the API key.